lib/mayaUsd/ufe/UsdUndoVisibleCommand.cpp

```diff
@@ -15,14 +15,10 @@
 //
 #include "UsdUndoVisibleCommand.h"
 
-#include "private/Utils.h"
-
 #include <mayaUsd/undo/UsdUndoBlock.h>
-#include <mayaUsdUtils/util.h>
 
 #include <pxr/usd/usdGeom/tokens.h>
-
-#include <ufe/log.h>
+#include <pxr/usd/usdGeom/xformCommonAPI.h>
```

Comment:
You sure we need this?  The common API doesn't feel like it has much to do with visibility...

Reply:
@ppt-adsk  yes it is needed for UsdGeomImageable.  It usually brings other stuff which we may need.  

But in this case, I don't think we need them. I will replace it with the below header if it is confusing.

#include <pxr/usd/usdGeom/imageable.h>